WordPress/src/main/java/org/wordpress/android/ui/stories/StoryComposerActivity.kt

```diff
@@ -138,11 +138,17 @@ class StoryComposerActivity : ComposeLoopFrameActivity(),
                     savedInstanceState.getSerializable(STATE_KEY_EDITOR_SESSION_DATA) as PostEditorAnalyticsSession
         }
 
-        editorMedia.start(site!!, this, STORY_EDITOR)
+        editorMedia.start(requireNotNull(site), this, STORY_EDITOR)
         postEditorAnalyticsSession?.start(null)
         startObserving()
     }
 
+    override fun onLoadFromIntent(intent: Intent) {
+        super.onLoadFromIntent(intent)
```

Comment:
we first try to handle the normal expected cases in the super class, and then try handling media intent data coming from other sources that the Stories library doesn't know of, but the host app does (for example, media Ids from the media section)